### DISCUSSION TITLE
posts/build-your-modern-neovim-config-in-lua/

### DISCUSSION THREAD

**Discussion by giscus** [m4xshen/m4xshen.dev#9] [Announcements]
# posts/build-your-modern-neovim-config-in-lua/

Learn how to build a modern Neovim configuration in Lua with options, mappings, autocmds, and plugins. Follow a structured approach to enhance your Neovim experience.

https://m4xshen.dev/posts/build-your-modern-neovim-config-in-lua/

**Comment by ghost** [comment]
Thank you for the awesome blog, I was able to configure my neovim setup just the way you described 🙌
